src/QueryEditor.tsx

```diff
@@ -35,18 +38,36 @@ export const QueryEditor: React.FC<Props> = ({ query, datasource, onChange, onRu
     handleQueryChange({ ...query, columns }, false);
   };
 
+  const loadTableOptions = _.debounce((query: string, cb?: LoadOptionsCallback<string>) => {
```

Comment:
Wrapping this in [debounce](https://lodash.com/docs/4.17.15#debounce) prevents us from firing too many queries while the user is still typing.